style.css

```diff
@@ -0,0 +1,118 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+    
+}
+
+body {
```

Comment:
Layout does not take all viewport height.

Fix flex property to let elements fit as expected.

![image](https://github.com/user-attachments/assets/3a03f67f-4679-454b-be9f-2784bb0da06b)